build/style_test.go

```diff
@@ -129,38 +129,72 @@ func TestStyle(t *testing.T) {
 
 	t.Run("TestEnvutil", func(t *testing.T) {
 		t.Parallel()
-		cmd, stderr, filter, err := dirCmd(pkg.Dir, "git", "grep", "-nE", `os\.(Getenv|LookupEnv)`, "--", "*.go")
-		if err != nil {
-			t.Fatal(err)
-		}
+		for _, tc := range []struct {
+			re       string
+			excludes []string
+		}{
+			{re: `\bos\.(Getenv|LookupEnv)\("COCKROACH`},
+			{
+				re: `\bos\.(Getenv|LookupEnv)\(`,
+				excludes: []string{
+					":!acceptance",
+					":!build/style_test.go",
```

Comment:
This isn't in `pkg`, is it? Why would it get picked up?

Reply:
I think `git grep` doesn't care about the directory you're in, or at least it doesn't when you specify `'*.go'`.